webrtc/go/.gitignore

```diff
@@ -0,0 +1,5 @@
+# we might ln -s these files to help during development,
+# but we don't want to commit these or the testground setup
+# will break (the mod file will overwrite the custom versions).
+/go.sum
```

Comment:
Is this a known issue for them, or expected? Because not having Go mod files to dos make IDE's not very happy in regular setups.

Reply:
Definitely a known issue - that text including the comment is ALL copied from an existing test (libp2p/test-plans/ping).

It was definitely a point of confusion for me.